plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from "react";
 import {
-  Table,
+  Table_Deprecated,
```

Comment:
I understand table was deprecated in 4.x release
Will you use non deprecated Table in the future?

Reply:
yes! next PR will have one.